CMSIS/NN/Source/SoftmaxFunctions/arm_softmax_s8.c

```diff
@@ -48,12 +89,118 @@ void arm_softmax_s8(const int8_t *input,
                     const int8_t diff_min,
                     int8_t *output)
 {
+#ifdef ARM_MATH_MVEI
+
+#define ACT_MIN (-128)
+#define ACT_MAX (127)
```

Comment:
Q7_MIN/MAX? Softmax shouldn't have weird activations attached to it since it is the last layer used for classification

Reply:
@giorgio-arenarm Sure.. i'll fix that.